README.md

```diff
@@ -60,7 +63,29 @@ Then navigate to https://github.com/elaskavaia/bga-sharedcode/pulls, click "New
         ".vscode",
         ".git",
         ".DS_Store",
-        "misc/"
-    ]
+        "misc/",
+        "node_modules",
+        "package-lock.json"
+    ],
+    "syncOption": {
```

Comment:
What does this do?

Reply:
https://github.com/liximomo/vscode-sftp/wiki/config#syncoption

deletes files that no longer are at the origin point and creates not files at destination